src/diffusers/models/attention_processor.py

```diff
@@ -18,6 +18,7 @@
 import torch
 import torch.nn.functional as F
 from torch import nn
+from einops import rearrange
```

Comment:
We like to refactor usage of `einops` `rearrange` to avoid the dependency. Let me know if you need assistance with that!